adcom1/app.go

```diff
@@ -114,6 +114,21 @@ type App struct {
 	//   Indicator of whether or not this is a paid app, where 0 = free, 1 = paid.
 	Paid int8 `json:"paid,omitempty"`
 
+	// Attribute:
+	//   inventorypartnerdomain
+	// Type:
+	//   string
+	// Description:
+	//   A domain to be used for inventory authorization in the case of inventory
+	//   sharing arrangements between an app owner and content owner. This field
+	//   is typically used by authorization crawlers to establish the domain of the
+	//   content owner, who has the right to monetize some portion of ad inventory
+	//   within the app. The content owner's domain should be listed in the app
+	//   owner's app-ads.txt file as an inventorypartnerdomain. Authorization for
+	//   supply from the inventorypartnerdomain will be published in the ads.txt
+	//   file on the root of that domain. Refer to the ads.txt 1.1 spec for more details.
+	InventoryPartnerDomain string `json:"inventorypartnerdomain,omitempty"`
+
```

Comment:
I'm confused as to why this was added to the adcom app object since this is an ORTB spec update PR. I also don't see this field defined it in the [Object: App](https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/master/AdCOM%20v1.0%20FINAL.md#object--app-) section of the AdCOM spec.